api/students/views.py

```diff
@@ -8,3 +8,19 @@
 class UserView(viewsets.ModelViewSet):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+class UserDetailView(generics.RetrieveUpdateDestroyAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
+
+    def get(self, request, *args, **kwargs):
+        try:
+            required_user = self.queryset.get(pk=kwargs["pk"])
+            return Response(UserSerializer(required_user).data)
```

Comment:
I think we should return `JSONResponse`.

Reply:
ek baar check kr le jaha se tune kiya hai.

Reply:
khud se kia hai

Reply:
![Screenshot 2019-10-27 at 12 58 35 PM](https://user-images.githubusercontent.com/31539812/67631236-92615000-f8b9-11e9-9338-26b676361c27.png)
JSON reponse hai

Reply:
UserSerialiser se JSON object aa rha hai